rke/k8s_rke_system_images.go

```diff
@@ -7670,6 +7670,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
+			Kubernetes:                "rancher/hyperkube:v1.22.13-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.87",
+			NginxProxy:                "rancher/rke-tools:v0.1.87",
+			CertDownloader:            "rancher/rke-tools:v0.1.87",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.87",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.17.4",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.17.4",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.17.4",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.21.1",
+			CalicoCNI:                 "rancher/calico-cni:v3.21.3-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.21.1",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.21.1",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.21.1",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.21.1",
+			CanalCNI:                  "rancher/calico-cni:v3.21.1-rancher1",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.21.1",
+			CanalFlannel:              "rancher/mirrored-flannelcni-flannel:v0.17.0",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.21.1",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.2.3.2.1d150da",
+			AciHostContainer:          "noiro/aci-containers-host:5.2.3.2.1d150da",
+			AciOpflexContainer:        "noiro/opflex:5.2.3.2.1d150da",
+			AciMcastContainer:         "noiro/opflex:5.2.3.2.1d150da",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.2.3.2.1d150da",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.2.3.2.1d150da",
+			AciGbpServerContainer:     "noiro/gbp-server:5.2.3.2.1d150da",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.2.3.2.1d150da",
+			PodInfraContainer:         "rancher/mirrored-pause:3.6",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.2.1-rancher1",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
+			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.1.1",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.5.1",
+			CoreDNS:                   "rancher/mirrored-coredns-coredns:1.8.6",
+			CoreDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.5",
+			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
+			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
+		},		
```

Comment:
Nit, but there are a couple of extra trailing tabs in this line (as well as at the end of the last line for all added blocks in this PR).

Reply:
removed the extra spaces